templates/code-of-conduct.html

```diff
@@ -0,0 +1,34 @@
+{% extends "base.html" %}
+{% load static %}
+
+{% block content %}
+
+<div class="row my-4">
+    <div class="col-12 text-center mb-3">
+        <h1>CODE OF CONDUCT</h1>
+        <p class="mt-2">Here at Code Institute we want to create a safe and welcoming environment for all hackathon participants.
+            It's important to keep open communication with your team throughout the hackathon, as suddenly dropping off
+            without letting them know can impact their progress and motivation, and may lead to incomplete tasks and
+            disorganization. By participating in this event, you agree to follow this code of conduct:</p>
+
+            <ol class="text-left">
+                <li>Be respectful and inclusive towards all other participants, facilitators, Hackteam, Staff or anyone else involved in the hackathon. Discriminatory language or behavior of any kind is not allowed.
+                </li>
+                <li>Only register if you can commit a minimum of 3 hours a day.</li>
+                <li>Keep in contact with your team. All project discussions should take place within the dedicated team Slack Channel. This way, our Hackathon Facilitators and Staff can stay up to date on your progress and offer support when needed. We want to make sure that everyone on the team is aware of all goings-on.
+                </li>
+                <li>We want to ensure that everyone on the team has a chance to participate and add their unique skills and perspectives. Let's work together to make sure everyone is included and valued. More experienced team members should keep in mind that some members may be less experienced, and may need extra support and guidance. By offering more time and assistance to those who may need it, we can ensure that everyone has an opportunity to grow their skills and contribute to the project. Remember, this is a team effort and we want to work together to achieve our goals rather than rushing ahead.
+                </li>
+                <li>If you need to drop off from the event before it ends, you must inform your team and the team facilitator. If you drop off without informing your team, you will be ineligible to participate in the next hackathon.
+                </li>
+                <li>If you witness or experience any behavior that violates this code of conduct, please report it immediately to your facilitator, the HackTeam or the Code Institute Community Team.</li>
+
+            </ol>
+            <p>By participating in a Code Institute Hackathon, you agree to abide by this code of conduct and any decisions made by Code Institute regarding violations of it.</p>
+    </div>
+</div>
+
+
+
+
+{% endblock %}
```

Comment:
Could you please add an extra line at EOF?

Reply:
resolved in 3774100